parachain/primitives/router/src/lib.rs

```diff
@@ -110,3 +178,129 @@ where
 		}
 	}
 }
+
+pub struct EthereumSovereignAccountConvertsFor<AccountId>(PhantomData<AccountId>);
```

Comment:
To be consistent with the naming of `GlobalConsensusParachainConvertsFor`:
```suggestion
pub struct GlobalConsensusEthereumContractConvertsFor<AccountId>(PhantomData<AccountId>);
```